core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -625,14 +635,35 @@ oppia.directive('conversationSkin', [
                     HintsAndSolutionManagerService.recordWrongAnswer();
 
                     PlayerTranscriptService.addNewResponse(feedbackHtml);
+                    var helpCardAvailable = false;
                     if (feedbackHtml &&
                         !ExplorationPlayerStateService.isInteractionInline(
                           $scope.activeCard.stateName)) {
+                      helpCardAvailable = true;
+                    }
+
+                    if (helpCardAvailable) {
                       $scope.$broadcast('helpCardAvailable', {
                         helpCardHtml: feedbackHtml,
                         hasContinueButton: false
                       });
                     }
+                    if (missingPrerequisiteSkillId) {
+                      ConceptCardBackendApiService.fetchConceptCard(
+                        missingPrerequisiteSkillId
+                      ).then(function(conceptCardBackendDict) {
+                        $scope.conceptCardPending = true;
+                        $scope.conceptCard =
+                          ConceptCardObjectFactory.createFromBackendDict(
+                            conceptCardBackendDict);
+                        if (helpCardAvailable) {
+                          $scope.$broadcast('helpCardAvailable', {
+                            helpCardHtml: feedbackHtml,
+                            hasContinueButton: true
```

Comment:
Just wondering, do we show anything like "Let's take a moment to look at this skill again" to signal the upcoming concept card?

Reply:
You mean signalling the learner that they are about to see a skill? I had asked Tony about this, and he told the feedback would be expected to have that information. So, I am showing the feedback and displaying Continue button to navigate to the skill.